core/server/master/src/main/java/alluxio/master/meta/MetaMasterFactory.java

```diff
@@ -49,11 +49,11 @@ public String getName() {
 
   @Override
   public MetaMaster create(MasterRegistry registry, JournalSystem journalFactory,
-      SafeModeManager safeModeManager, long startTimeMs) {
+      SafeModeManager safeModeManager, long startTimeMs, int port) {
```

Comment:
just curious, do you think it is required to have `port` on the signature (this increases the footprint of this PR by a lot). possible to `Configuration.getInt(PropertyKey.MASTER_RPC_PORT)` inside the creation of `MasterContext`?

Reply:
Just FYI the journal backup/restore PR I'm working on does some refactoring to allow new fields to be added to `MasterContext` without much of a footprint: https://github.com/Alluxio/alluxio/pull/7318/files

I don't have a strong opinion either way for whether to use `Configuration` vs `MasterContext` for now. Once that refactor happens it will be easier to use `MasterContext` for passing the port

Reply:
ack